packages/mdc-form-field/_mixins.scss

```
@@ -66,4 +66,22 @@
       }
     }
   }
+
+  .mdc-form-field--space-between {
+    @include feature-targeting-mixins.targets($feat-structure) {
+      justify-content: space-between;
+    }
+
+    // stylelint-disable-next-line selector-max-type
+    > label {
+      @include feature-targeting-mixins.targets($feat-structure) {
+        margin: initial;
```

Well I thought we were close. IE11 doesn't support `initial`, so this needs to be `0` 😣

```suggestion
        margin: 0;
```